Data/Scripts/014_Pokemon/001_Pokemon-related/003_Pokemon_Sprites.rb

```diff
@@ -187,13 +187,20 @@ def createFusionIcon()
     icon1 = AnimatedBitmap.new(GameData::Species.icon_filename(headPoke))
     icon2 = AnimatedBitmap.new(GameData::Species.icon_filename(bodyPoke))
 
```

Comment:
```
    directory_name = "Graphics/Pokemon/FusionIcons"
    Dir.mkdir(directory_name) unless File.exists?(directory_name)
```
needs to be added, to check if directory exist, and create it if it doesn't exist. Game will crash without this code if the dir doesn't exist !